setup.py

```diff
@@ -4,15 +4,18 @@
 
 extras_require = {
     'test': [
-        "pytest==3.2.2",
-        "pytest-xdist"
+        "mypy-extensions>=0.4.1",
```

Comment:
This will have to go in the core `install_requires`, otherwise the import of the `TypedDict` type will fail on normal installation.